packs/go/charts/Makefile

```diff
@@ -2,7 +2,7 @@ CHART_REPO := http://jenkins-x-chartmuseum:8080
 CURRENT=$(pwd)
 NAME := REPLACE_ME_APP_NAME
 OS := $(shell uname)
-VERSION := $(shell cat ../../VERSION)
+RELEASE_VERSION := $(shell cat ../../VERSION)
```

Comment:
For some reason, the `go` pack is subtly different from all the rest. Starting to normalize that here. The other difference is `$(NAME)` vs. `REPLACE_ME_APP_NAME` below, though I was unsure whether it was proper to also normalize that.